package.json

```diff
@@ -31,6 +31,7 @@
     "react-collapse": "^4.0.3",
     "react-dom": "16.4.1",
     "react-motion": "^0.5.2",
+    "react-router-dom": "^4.3.1",
```

Comment:
Same concern as the previous PR: https://github.com/ClimateWatch-Vizzuality/climate-watch-components/pull/7#discussion_r207223268